uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/apis/work-pattern-api.js

```diff
@@ -10,18 +11,19 @@ define([
     return api.extend({
 
       /**
-       * This method returns the calendar for a specific period, as a list of days and their type
+       * This method returns the calendar for the given contact(s) and period,
+       * as a list of days and their type
        *
-       * @param {string} contactId The ID of the Contact
-       * @param {string} periodId The ID of the Absence Period
-       * @param {object} params
+       * @param {string/int/Array} contactId can be also an array for multiple contacts
+       * @param {string/int} periodId The ID of the Absence Period
+       * @param {object} params additional parameters
        * @return {Promise} Resolved with {Array} All calendar records
        */
       getCalendar: function (contactId, periodId, params) {
         $log.debug('WorkPatternAPI.getCalendar', contactId, periodId, params);
 
         return this.sendGET('WorkPattern', 'getcalendar',  _.assign({}, params, {
-          contact_id: contactId,
+          contact_id: _.isArray(contactId) ? { "IN": contactId } : contactId,
```

Comment:
We can always use `{ "IN": contactId }`, if it is a single value, we will get value for the same.